WordPressKit/WordPressKit/MediaServiceRemoteREST.h

```diff
@@ -6,7 +6,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface MediaServiceRemoteREST : SiteServiceRemoteWordPressComREST <MediaServiceRemote>
 
-- (RemoteMedia *)remoteMediaFromJSONDictionary:(NSDictionary *)jsonMedia;
++ (RemoteMedia *)remoteMediaFromJSONDictionary:(NSDictionary *)jsonMedia;
```

Comment:
Will be nice to have some documentation for both methods now that they are public.

Reply:
Good point @SergioEstevao. I will add it.